integration-tests/src/bin/gen_blockchain_data.rs

```diff
@@ -156,240 +49,162 @@ async fn main() {
         "Transfer 0".to_string(),
         receipt.block_number.unwrap().as_u64(),
     );
+    info!("- done(height: {:?})", receipt.block_number);
```

Comment:
```suggestion
    info!("- Done(height: {:?})", receipt.block_number);
```
Can you apply all the changes to done? I suggest this change because we use "Doing" capitalized.